pkgs/by-name/zp/zpp/package.nix

```diff
@@ -0,0 +1,3 @@
+{ python3Packages }:
```

Comment:
Why not place this in all-packages.nix?

Reply:
Isn't that the new recommended way ?
If not I can for sure put it back in `all-packages.nix` .

Reply:
@SuperSandro2000, hexa advised me favor the `by-name` approach.

Reply:
> Isn't that the new recommended way ?

I thought that is for new packages like pdi, not for "aliases", short hands or multi version packages where I think this exactly falls into it.

Reply:
Yes, I thought too. This is why I wanted to ask hexa, and he said "just use the by-name approach, dealing with all-packages.nix is too annoying".

Reply:
Ok, I will go with the current implementation then.

While I was looking to create a PR to document this choice, I realized that this is already the recommended process: https://nixos.org/manual/nixpkgs/unstable/#topythonapplication-function

Reply:
Thank you once again for your careful review ! I appreciate :)

Reply:
Glad that I am just hearing now from all the changes in the python language document.